Modules/overlapped.c

```diff
@@ -612,6 +639,31 @@ Overlapped_dealloc(OverlappedObject *self)
     SetLastError(olderr);
 }
 
+static PyObject*
+unparse_address(LPSOCKADDR Address, DWORD Length) {
+    // An IPv6 address has a maximum length of 39 characters
+    char AddressString[47];
+    unsigned int port;
+    PVOID pSinAddr;
+
+    switch(Address->sa_family) {
+    case AF_INET:
+        port = ntohs(((SOCKADDR_IN*)Address)->sin_port);
+        pSinAddr = &((SOCKADDR_IN*)Address)->sin_addr;
+        break;
+    case AF_INET6:
+        port = ntohs(((SOCKADDR_IN6*)Address)->sin6_port);
+        pSinAddr = &((SOCKADDR_IN6*)Address)->sin6_addr;
+        break;
+    default:
+        return SetFromWindowsErr(ERROR_INVALID_PARAMETER);
+    }
+
+    inet_ntop(Address->sa_family, pSinAddr, AddressString, 47);
```

Comment:
Please check `inet_ntop` return value, an exception should be raised if NULL is returned.